drivers/azure/azureutil/azureutil.go

```diff
@@ -586,6 +593,8 @@ func getOSDisk(name string, osDiskBlobURL string, isManaged bool, storageType st
 			DiskSizeGB: to.Int32Ptr(diskSize),
 		}
 	} else {
+		osDiskBlobURL := osDiskStorageBlobURL(account, name)
+		log.Debugf("OS disk blob will be placed at: %s", osDiskBlobURL)
```

Comment:
If account is nil this will say `OS disk blob will be placed at:`

Reply:
since its debug, im ok with it